x-pack/platform/plugins/private/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_view.tsx

```diff
@@ -515,49 +511,40 @@ export const IndexDataVisualizerView: FC<IndexDataVisualizerViewProps> = (dataVi
       paddingSize="none"
     >
       <EuiPageTemplate.Section>
-        <EuiPageTemplate.Header data-test-subj="dataVisualizerPageHeader" css={dvPageHeader}>
-          <EuiFlexGroup
-            data-test-subj="dataViewTitleHeader"
-            direction="row"
-            alignItems="center"
-            css={{ padding: `${euiTheme.euiSizeS} 0`, marginRight: `${euiTheme.euiSize}` }}
-          >
-            <EuiTitle size={'s'}>
-              <h2>{currentDataView.getName()}</h2>
-            </EuiTitle>
-            <DataVisualizerDataViewManagement currentDataView={currentDataView} />
-          </EuiFlexGroup>
-
-          {isWithinLargeBreakpoint ? <EuiSpacer size="m" /> : null}
-          <EuiFlexGroup
-            alignItems="center"
-            justifyContent="flexEnd"
-            gutterSize="s"
-            data-test-subj="dataVisualizerTimeRangeSelectorSection"
-          >
-            {hasValidTimeField ? (
-              <EuiFlexItem grow={false}>
-                <FullTimeRangeSelector
-                  frozenDataPreference={frozenDataPreference}
-                  setFrozenDataPreference={setFrozenDataPreference}
-                  dataView={currentDataView}
-                  query={undefined}
-                  disabled={false}
-                  timefilter={timefilter}
-                />
-              </EuiFlexItem>
-            ) : null}
-            <EuiFlexItem grow={false}>
-              <DatePickerWrapper
-                isAutoRefreshOnly={!hasValidTimeField}
-                showRefresh={!hasValidTimeField}
-                width="full"
-                needsUpdate={queryNeedsUpdate}
-                onRefresh={handleRefresh}
+        <EuiPageTemplate.Header
+          data-test-subj="dataVisualizerPageHeader"
+          css={dvPageHeader}
+          pageTitle={
+            <>
+              {currentDataView.getName()}
+              {/* TODO: This management section shouldn't live inside the header */}
```

Comment:
Are you planning on doing this in this PR?

Reply:
I think we can tackle it, but it would be nice to find a different place for this menu to reside. Otherwise, it's being wrapped within an <h1> tag, which doesn't seem right.

If @joana-cps suggests a better location for the button, we can definitely move it outside the header.

Reply:
I agree that we need to find a different place for this menu. We can do this in a separate issue.